### ISSUE TITLE
Allow implicit `Unknown` type for ignored arguments of a function. (reportUnknownParameterType)

### ISSUE THREAD

**Issue by pawelrubin** [microsoft/pyright#699]
**Is your feature request related to a problem? Please describe.**
Sometimes functions arguments are not used, hence we ignore them via `_` or `*_`. 

Consider this piece of code:
```python
def foo(a: int, b: int, *_) -> int:
    return max(a, b)
```

Pyright output when hovering over `*_`:
```
_
(parameter) _: Unknown
Type of parameter "_" is unknownPyright (reportUnknownParameterType)
```
**Describe the solution you'd like**
It would be awesome if Pyright would allow for not specifying the type of ignored arguments.

**Comment by erictraut** [comment]
Good suggestion. This will be implemented in the next version of pyright.

**Comment by erictraut** [comment]
This is now fixed in Pyright 1.1.40, which I just published.
